oak/server/rust/oak_loader/src/main.rs

```diff
@@ -199,8 +200,14 @@ fn main() -> anyhow::Result<()> {
         read_to_string(&opt.grpc_tls_private_key).context("could not read gRPC TLS private key")?;
     let grpc_tls_certificate =
         read_to_string(&opt.grpc_tls_certificate).context("could not read gRPC TLS certificate")?;
-    let root_tls_certificate =
-        read_to_string(&opt.root_tls_certificate).context("could not read root TLS certificate")?;
+    let root_tls_certificate = match &opt.root_tls_certificate {
+        Some(certificate_path) if cfg!(feature = "oak_debug") => {
+            read_to_string(certificate_path).context("could not read root TLS certificate")?
+        }
+        _ => std::str::from_utf8(include_bytes!("certs/roots.pem"))
```

Comment:
Can it also show an error if we specify a gRPC cert parameter without enabling `debug`?

Reply:
Done.